app/frontend/wallet/helpers/tokenFormater.ts

```diff
@@ -7,13 +7,12 @@ import {TokenObject} from '../backend-types'
 export const arraySum = (numbers: Array<number>): number =>
   numbers.reduce((acc: number, val) => acc + val, 0)
 
-export const formatToken = (token: TokenObject, multiplier: number = 1): Token => ({
-  policyId: token.policyId,
-  assetName: token.assetName,
-  quantity: multiplier * parseInt(token.quantity, 10),
+export const formatToken = (token: TokenObject): Token => ({
```

Comment:
this is really a misleading name - until I read the code I though it formats the token into its string representation. Perhaps `parseToken` would be more accurate

Reply:
I agree :+1:

Reply:
Fixed.